src/topUi.ml

```diff
@@ -92,10 +92,7 @@ let display_stdout top response =
         let line =
           if offset >= 1024 then ""
           else if offset + String.length line <= 1024 then line
-          else
-            let s = String.sub line 0 (1024 - offset) in
-            String.blit "..." 0 s (1024 - offset - 3) 3;
-            s
+          else (String.sub line 0 (1024 - offset - 3) ^ "...")
```

Comment:
Not really an issue here, but you're doing one more 1024 bytes allocation than before :)

Reply:
Yes. I was thinking about going through the `Bytes` module, but I decided, that it not worth the complexity